themes/digital.gov/src/js/glossary.js

```diff
@@ -0,0 +1,251 @@
+  if (e.target.matches(`.${cssClasses.term}`)) {
+    const termNumber = e.target.getAttribute("aria-controls");
+    e.target.setAttribute(
+      "aria-expanded",
+      e.target.getAttribute("aria-expanded") === "true" ? "false" : "true"
+    );
+    glossaryBody
+      .querySelectorAll(`#${termNumber}, #${termNumber}-icon`)
+      .forEach((definition) => definition.toggleAttribute("hidden"));
+  }
+}
+
+function initializeEventListeners() {
+  /**
+   * Open glossary when toggle button is clicked
+   */
+  toggleButtons.forEach((button) => {
+    button.addEventListener("click", () => showGlossary(button));
+  });
+
+  /**
+   * Close glossary on escape keypress
+   * @param {event} e to handle keycode lookup when "escape" is pressed
+   */
+  document.body.addEventListener("keyup", (e) => {
+    if (e.keyCode === KEYCODE_ESC) {
+      hideGlossary();
+    }
+  });
+
+  /**
+   * Close glossary when clicking outside of aside (and not on toggle button)
+   * @param {event} e to handle click on document body
+   */
+  document.body.addEventListener("click", (e) => {
+    const buttons = Array.from(toggleButtons);
+    if (!buttons.includes(e.target)) {
+      if (!glossaryBody.contains(e.target)) {
+        hideGlossary();
+      }
+    }
+  });
+
+  /**
+   * Close glossary when close button is clicked
+   */
+  closeButton.addEventListener("click", hideGlossary);
+
+  /**
+   * When the first item (close button) is reached, refocus the top of glossary
+   * @param {event} e to handle tab key press (changing focus)
+   */
+  closeButton.addEventListener("keydown", (e) => {
+    if (e.keyCode === KEYCODE_TAB) {
+      search.focus();
+    }
+  });
+
+  /**
+   * When the last element in the list is reached, focus the top of the glossary
+   * @param {event} e to handle tab key press (changing focus)
+   */
+  listElement.lastChild.firstElementChild.addEventListener("keydown", (e) => {
+    if (e.keyCode === KEYCODE_TAB) {
+      search.focus();
+    }
+  });
+
+  /**
+   * Filter glossary terms on search
+   */
+  search.addEventListener("input", () => {
+    if (list.filtered) {
+      list.filter();
+    }
+  });
+
+  /**
+   * When a glossary term is clicked, toggle the definition expansion
+   */
+  glossaryBody.addEventListener("click", (e) => handleTermClick(e));
+}
+
+/**
+ * Initialize the glossary DOM elements
+ * Fetch the glossary terms from the "glossaryPath" variable set in baseof.html
+ * Generate list markup, initialize new list, and initialize event listeners
+ * @param {string} path relative path to the json file with terms
+ */
+
+async function initializeGlossary(path) {
+  initializeElements();
+
+  const response = await fetch(path);
+  if (!response.ok) return;
+  const terms = await response.json();
+
+  generateListMarkup(terms);
+
+  initializeList();
+
+  initializeEventListeners();
+}
+
+/* eslint-disable no-undef */
+if (glossaryBody) {
+  if (glossaryPath) {
+    initializeGlossary(glossaryPath);
+  }
+}
```

Comment:
Couldn't you combine these?

```suggestion
if (glossaryBody && glossaryPath) {
  initializeGlossary(glossaryPath);
}
```